pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: shelf_web_socket
-version: 0.2.4+1
+version: 0.2.5
```

Comment:
Bump this to 0.3.0 – we usually consider null safe migrations to be breaking changes

Reply:
Resolving dependencies...
Because no versions of test match >1.16.3 <2.0.0 and test 1.16.3 depends on shelf_web_socket ^0.2.0, test ^1.16.3 requires shelf_web_socket ^0.2.0.
So, because shelf_web_socket depends on test ^1.16.3 and shelf_web_socket is 0.3.0, version solving failed.

Reply:
if you open https://github.com/dart-lang/test/blob/master/pkgs/test/pubspec.yaml, you can see such dependencies
...
   shelf: '> = 0.7.0 <2.0.0'
   shelf_packages_handler: "> = 1.0.0 <3.0.0"
   shelf_static: ^ 0.2.6
   shelf_web_socket: ^ 0.2.0
   source_span: ^ 1.8.0
...
This means that the test package depends on the shelf_web_socket package. And shelf_web_socket depends on test :)

Reply:
Yup. We'll have to bump the test version first. This is a bit of a pain, but...sigh...common

Reply:
We should add dependency overrides here instead of bumping package:test first:

```
dependency_overrides:
  test: ^1.16.0
```

You may need to add some others.

Reply:
Also consider bumping the version to `1.0.0`

Reply:
done